lib/gitlab/commits_calendar.rb

```diff
@@ -21,5 +21,13 @@ def initialize(repositories, user)
         @timestamps[timestamp] = commits if timestamp
       end
     end
+
+    def starting_year
+      (Time.now - 1.year).strftime("%Y")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.